sbin/devd/devd.cc

```diff
@@ -1107,6 +1107,14 @@ event_loop(void)
 			err(1, "select");
 		} else if (rv == 0)
 			check_clients();
+		if (FD_ISSET(stream_fd, &fds))
+			new_client(stream_fd, SOCK_STREAM);
+		/*
+		 * Aside from the socket type, both sockets use the same
+		 * protocol, so we can process clients the same way.
+		 */
```

Comment:
If you are already touching the code...
Maybe move the if that's above the comment to just after the comment. The comment makes more sense about what it applies to and the code looks neater IMHO. I think the original commit of the comment had it out of place. I wouldn't put a blank line betwee the two ifs.

Otherwise, this change looks good.